cfme/automate/explorer/method.py

```diff
@@ -260,6 +261,16 @@ class MethodAddView(AutomateExplorerView):
     embedded_method = EntryPoint(locator='//*[@id="automate-inline-method-select"]//button',
                                  tree_id="treeview-entrypoint_selection")
 
+    # Ansible Tower Workflow Template
+    ansible_tower_workflow_template_name = Input(name="name")
+    ansible_tower_workflow_template_display_name = Input(name="display_name")
+
+    # Ansible Tower Job Template
+    ansible_tower_job_template_name = Input(name="name")
```

Comment:
Q: Any `id` to distinguish them from the widgets of the Tower Workflow Templates?